service/flannelconfig/v2/resource/flanneldv1/create.go

```diff
@@ -7,7 +7,7 @@ import (
 	"k8s.io/api/extensions/v1beta1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 
-	"github.com/giantswarm/flannel-operator/service/keyv2"
+	"github.com/giantswarm/flannel-operator/service/flannelconfig/v2/keyv2"
```

Comment:
Do you want to to live it `flannelconfig/v2/keyv2` or rename in followup PR?

Reply:
The package naming will change in the next steps.